README.md

```diff
@@ -4,6 +4,17 @@ This repository contains unofficial patterns, sample code, or tools to help deve
 
 # Fauna Schema Migrate
 
+## Deprecation Notice
+
+**This project is deprecated and no longer in development** 
+
+Fauna has introduced a suite of officially supported schema languages and tooling which fully replace Fauna Schema Migrate. These are the [Fauna Schema Language](https://docs.fauna.com/fauna/current/cookbook/advanced/fsl/get_started_fsl) (FSL),  [Fauna Command Line Interface](https://docs.fauna.com/fauna/current/tools/shell/) (Fauna CLI), and [GitHub and GitLab integrations](https://fauna.com/blog/supercharge-your-devops-workflow-with-fauna-schema-language). Combined, these features allow you to apply DevOps priciples to database development, and incorporate schema-level changes into broader CI/CD workflows.
+
+**The Fauna Schema Language** is a declarative language for expressing your entire database schema as a set of .fsl files. FSL files are intended to be managed in the same fashion as users manage their application code. That is, FSL files can be managed in common version revision control systems, where they can be shared, peer reviewed, version tracked, and where changes are automated into users' continuous integration and continuous deployment pipelines.
```

Comment:
- Hyphenate "peer-reviewed".
- Should read "peer-reviewed **and** version tracked,"
- Change "fashion as" to "way that".
- "version revision" is kind of redundant.